src/db/models/user.ts

```diff
@@ -0,0 +1,28 @@
+import { Sequelize } from 'sequelize';
+
+const Users = (sequelize: Sequelize, DataTypes: any) => {
+  const User = sequelize.define('User', {
+    id: {
+      unique: true,
+      primaryKey: true,
+      type: DataTypes.STRING,
```

Comment:
This model says that the `id` is a string instead of a number

Reply:
This is just a dummy and will be removed when we are implementing the user functionality, this is just to ensure the Sequelize integration is actually working, beside the id should be a string and not a number, I am advocating we use a fancy id, `-LXcDefU_UyicVreTyrJiY`

Reply:
I think anyone is fine. I only mentioned it because I saw `sequelize.NUMBER` for `id` in the migration file.